src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -141,6 +161,32 @@ public void processProjectProperties(List<Row> rows)
          properties.setProjectTitle(row.getString("proj_short_name"));
          properties.setDefaultTaskType(TASK_TYPE_MAP.get(row.getString("def_duration_type")));
          properties.setStatusDate(row.getDate("last_recalc_date"));
+         properties.setFiscalYearStartMonth(row.getInteger("fy_start_month_num"));
+         // cannot assign actual calendar yet as it has not been read yet
+         m_defaultCalendarID = row.getInteger("clndr_id");
+      }
+   }
+
+   /**
+    * Process schedule options.
+    *
+    * @param rows schedule options data.
+    */
+   public void processScheduleOptions(List<Row> rows)
+   {
+      if (rows.isEmpty() == false)
+      {
+         Row row = rows.get(0);
+
+         Map<String, Object> customProperties = m_project.getProjectProperties().getCustomProperties();
```

Comment:
You can simply create the `Map` here and pass it to `setCustomProperties`. The reader is responsible for populating all of the attributes, so this value won't have been set elsewhere.